src/rpc/handlers/AccountInfo.cpp

```diff
@@ -112,27 +113,28 @@ AccountInfoHandler::process(AccountInfoHandler::Input const& input, Context cons
             signerList.push_back(sleSigners);
         }
 
-        return Output(
-            lgrInfo.seq,
-            ripple::strHex(lgrInfo.hash),
-            sle,
-            isDisallowIncomingEnabled,
-            isClawbackEnabled,
-            isTokenEscrowEnabled,
-            ctx.apiVersion,
-            signerList
-        );
+        return Output{
+            .ledgerIndex = lgrInfo.seq,
+            .ledgerHash = ripple::strHex(lgrInfo.hash),
+            .accountData = sle,
+            .isDisallowIncomingEnabled = isDisallowIncomingEnabled,
+            .isClawbackEnabled = isClawbackEnabled,
+            .isTokenEscrowEnabled = isTokenEscrowEnabled,
+            .apiVersion = ctx.apiVersion,
+            .signerLists = signerList
+        };
```

Comment:
Can we only do one return in the end maybe?
As far as I see, the only difference is `.signerLists` which we can probably move out of this `if` and in the end write ternary.